src/Integration.Vsix.UnitTests/Analysis/AnalysisStatusNotifierTests.cs

```diff
@@ -141,6 +142,24 @@ public void AnalysisFailed_LogToOutputWindow()
             logger.OutputStrings.Count.Should().Be(1);
         }
 
+        [TestMethod]
+        public void AnalysisFailed_AggregateException_LogToOutputWindow()
```

Comment:
We had a special case in EslintBridgeHttpWrapper that collected the ex.Message of the inner exceptions.
However, in AnalysisStatusNotifier we always outputted ex.ToString(), and that works also for aggregate exceptions.

I think we that in general we might want to handle specific exceptions in JavascriptAnalyzer. For instance, if the process was killed, an WebException is thrown with message "Unable to connect to the remote server", and it contains an inner exception "SocketException: No connection could be made because the target machine actively refused it", which is a lot more useful from a troubleshooting standpoint. So we should probably catch WebExceptions and report the ex.Message of the underlying exception.

For now I left it with toString() and we could change to Message if we want to.